Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE_description.md

```diff
@@ -8,4 +8,18 @@ This integration was integrated and tested with version 1.0 of Prisma SASE API.
 
 
 ## Prisma SASE API
-[Prisma SASE API](https://pan.dev/sase).
\ No newline at end of file
+[Prisma SASE API](https://pan.dev/sase).
+
+## Required Permissions
+
+To use the integration commands, ensure the ***Prisma Access & NGFW Configuration*** has one of the following roles assigned:
```

Comment:
```suggestion
To use the integration commands, ensure the **Prisma Access & NGFW Configuration** has one of the following roles assigned:
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.